src/Ucommerce.Avenue.Umbraco/Search/AvenueProductIndexDefinition.cs

```diff
@@ -12,20 +11,10 @@ public AvenueProductIndexDefinition() : base()
             this.Field(p => p["CollarSize"], typeof(string));
             this.Field(p => p["ShoeSize"], typeof(string));
             this.Field(p => p["Colour"], typeof(string));
-
+            this.PricesField(p => p.UnitPrices.ToString());
```

Comment:
Hmmm....
We don't want to limit the number of price dictionaries that a customer could add. Already, we talked about adding YourPrice in addition to ListPrice to the default product index. It's very likely to happen.
Can there be more than one PricesField?

Reply:
Also, what's the `.ToString()` doing there. Looks a bit odd.

Reply:
you are right, I forgot about this